balancer/balancer.go

```diff
@@ -45,54 +48,49 @@ type balanceEntity interface {
 }
 
 type peer struct {
-	hostname    string
-	entities    []balanceEntity
-	leaderCount int
-	rebalance   int
+	name     string
+	entities []balanceEntity
+	offset   int
+}
+
+const SERVER_VERSION = "v2.11.0"
+
+func checkServerVersion(nc *nats.Conn) error {
+	version := nc.ConnectedServerVersion()
```

Comment:
Rather use this and look for api level 1

https://github.com/nats-io/jsm.go/blob/e2f9c88cac4703f1149605d607d633b5bdb8662a/manager.go#L105


We don’t know what version the meta leader is - the connection machine might be different. 

API level checks talk to the machine that is jetstream leader so more reliable. 

1 means 2.11